srcpkgs/libxslt/template

```diff
@@ -28,6 +26,12 @@ post_configure() {
 	fi
 }
 
+post_build() {
+	# xsltproc manpage depends on xsltproc, this remakes it after
+	# libxslt is build
+	make -C doc XSLTPROC=${wrksrc}/xsltproc/xsltproc xsltproc.1
```

Comment:
Any idea why this works for cross?

Reply:
_That's a really good question._

Reply:
I didn't think of that. It (of course) doesn't work for cross, it just ignores errors while generating the manpage. I have included a premade manpage for use in the cross compiled version.